### ISSUE TITLE
Deltas?

### ISSUE THREAD

**Issue by ArtemGr** [ArtemGr/Sidekick#6]
I've heard that Fantasy Flight Games (Warhammer?) has those d100 rolls where you calculate a **delta** from the target number. The larger the delta, the greater the degree of success is. Likewise for failure.

Need more info on this. Also need to think of a way to print the delta without disrupting the other games.

**Comment by TheRandomNPC** [comment]
Yes, it's mostly the Warhammer settings. You roll a d100 and if it is under the target number, it's a success. Every full 10 points is a degree. Example, target number is 47, roll is 25, that's a success with 2 degrees. If the roll was a 44, it would still be a success, but have no degrees. (Also, I was trying to get a copy of Sidekick for desktop use, but all I found was a Readme and License).

**Comment by ArtemGr** [comment]
@TheRandomNPC 

Sidekick works from a server, it's a part of a larger code base. Repackaging it for desktop use is so far a non-goal.

So, are we talking https://en.wikipedia.org/wiki/Dark_Heresy_(role-playing_game) ?

I might try and skim through the rule book but I don't want to be picking the wrong one.

**Comment by TheRandomNPC** [comment]
I understand, and yes, though I believe Deathwatch and Rogue Trader use the same system.

**Comment by ArtemGr** [comment]
Cool, thanks. I'll look into it.

**Comment by dumire** [comment]
There's a fan-made Elder Scrolls RPG being worked on that uses a similar system but (currently) counts degrees of success differently. They subtract tens digit of the roll from the tens digit of the target and add 1. So for the example above for Warhammer, target 47 roll 25 would be 3 DoS, while target 47 roll 45 would be 1 DoS.

I think the Warhammer way is the "normal" way (it's a published game), but just wanted to bring it up for consideration.

Link to UESRPG files: [https://docs.google.com/document/d/1pTgTN2aJUoY95JtquowagfUJLL7tCQYhzJKcCAcbvio/edit](url)
